mypyc/irbuild/ll_builder.py

```diff
@@ -255,6 +284,190 @@ def other() -> Value:
 
     # Calls
 
+    def _construct_varargs(self,
+                           args: Sequence[Tuple[Value, ArgKind, Optional[str]]],
+                           line: int,
+                           *,
+                           has_star: bool,
+                           has_star2: bool) -> Tuple[Optional[Value], Optional[Value]]:
+        """Construct *args and **kwargs from a collection of arguments
+
+        This is pretty complicated, and almost all of the complication here stems from
```

Comment:
Thanks for writing such a detailed explanation! This really made it easier to figure what is going on.

Can you briefly also document `has_star` and `has_star2`?